scheduler_test.go

```diff
@@ -145,6 +145,12 @@ func TestScheduler_Every(t *testing.T) {
 		}
 		s.Stop()
 		assert.Equal(t, 6, counter)
+
+		_, err = s.Every(-1 * time.Millisecond).Do(func() {
+			// do nothing
+		})
+
+		require.EqualError(t, err, ErrInvalidInterval.Error())
```

Comment:
ah, the error is wrapped. So it's got another one around it. I think using `require.ErrorIs` should do it

Reply:
thanks, I actually removed these checks as I realized there was already a bad interval test case, so I added cases to that and switched it to use contain.